lib/render/leaderboard.php

```diff
@@ -613,3 +613,44 @@ function getGlobalRankingData($lbType, $sort, $date, $user, $friendsOf = null, $
     }
     return $retVal;
 }
+
```

Comment:
Is there a reason why were' putting this new code in `leaderboard.php`? Shouldn't it be in its own file?

Reply:
You're right, moved it to database/mastery.php

Reply:
All of the other SiteAward functionality is in user.php, but this isn't really user specific. I'm fine with this being its own file, but it would probably be acceptable to put it into activity.php too.